### ISSUE TITLE
Merge of Sensitive Information and Credentials Transport in Clear Text

### ISSUE THREAD

**Issue by ThunderSon** [OWASP/wstg#598]
**What would you like to happen?**
Merge ATHN-01 and CRYP-03 as they represent in their entirety the same thing, just different terms (credentials versus sensitive info).

**Comment by kingthorin** [comment]
Agreed in 2013/2014 before the drive toward all https all the time it made sense for them to be split. Today, however, it probably makes sense for credentials to just be another type of "sensitive information". So lumping them both together makes sense to me. I'd be fine with ATHN-01 being chopped and it's content merged into CRYP-03. (A lead-in (0th page) for 4.4 could be added that clarifies this.)

**Comment by manindar-mohan** [comment]
i will take this up.

**Comment by manindar-mohan** [comment]
@kingthorin, is this issue closed?

**Comment by kingthorin** [comment]
I don't think so. 
To be sure you'd have to check the two sections.
I'd be pretty surprised if it was done without a PR linked here in the history though.
